src/as-validator.c

```diff
@@ -1689,6 +1689,12 @@ as_validator_check_release (AsValidator *validator, xmlNode *node, AsFormatStyle
 		/* Neither timestamp, nor date property exists */
```

Comment:
nitpick: Please move that comment past the `if (timestamp == NULL)` condition:
```c
if (timestamp == NULL) {
    /* Neither timestamp, nor date property exists */
    as_validator_add_issue (validator, node, "release-time-missing", "date");
} else {
    ....
```

Reply:
You missed this one, I think ;-)

Reply:
Sorry